codegen-on-oss/codegen_on_oss/context_server/server.py

```
@@ -0,0 +1,347 @@
+        # Create and run the agent
+        agent = CodeAgent(
+            codebase=harness.codebase,
+            tags=harness.tags,
+            metadata=harness.metadata,
+        )
+        
+        result = agent.run(prompt=request.prompt)
+        
+        # Get the diff if there were changes
+        diff = harness.codebase.get_diff()
+        
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)